_inc/client/components/forms/index.jsx

```diff
@@ -89,9 +89,17 @@ export class FormTextInput extends React.Component {
 			'is-valid': this.props.isValid,
 		} );
 
+		const filteredProps = {};
+		for ( const key in this.props ) {
+			if ( [ 'isError', 'isValid', 'selectOnFocus' ].includes( key ) ) {
+				continue;
+			}
+			filteredProps[ key ] = this.props[ key ];
+		}
+
 		return (
 			<input
-				{ ...this.props }
+				{ ...filteredProps }
```

Comment:
This change prevents the props used for rendering from being passed to the `<input>` and printing errors to the console.